### PR TITLE
 Added Contact form

### PR DESCRIPTION
**Pull request by ahmadsarfrazx**
- Added h1 tag in the Contact form
- Added description in the Contact form
- Added the contact form
- Added styles to it, to make it look like a Figma template
- Added a way to receive the Forms submitted by others
- Removed the icon bar from the bottom of the contact page
- Increase font-weight to 400 to match the Figma template
- Added background image in the image folder

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -351,3 +351,97 @@ h2 {
...
+  flex-direction: column;
+  background-image: url(assets/images/background2.png);
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-size: 70%;
+  background-position-x: right;
+  background-position-y: 20px;
+  border-radius: 90px 0 0 0;
+  padding-top: 103px;
+}
```
Comment (levy002) [inline style.css]: - [ ] Kindly make sure the background image `assets/images/background2.png` you are targeting is stored in your images folder.

    - ### Your design
   image

  - ### Figma design
 
   image

File: style.css [hunk 2]
```diff
@@ -351,3 +351,97 @@ h2 {
...
+
+.p-contact {
+  font-style: normal;
+  font-weight: 300;
+  font-size: 18px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  padding: 24px;
+}
```
Comment (levy002) [inline style.css]: - [ ] Kindy make sure to use to correct typography on this paragraph in order to meet the provided Figma design. :wink: 

   - ### Your design
    image

   - ### Figma design
    image

File: index.html [hunk 3]
```diff
@@ -252,6 +252,29 @@ <h2 class="heading-2">About <br />Myself</h2>
...
+          <input type="email" name="_replyTo" class="inp-form" id="email" placeholder="Email" required />
+          <label for="message"></label>
+          <textarea name="message" class="text-area" cols="30" rows="10" maxlength="500"
+            placeholder="Write your message here" required id="message"></textarea>
+          <small id="message-success" class="hidden">From Submitted</small>
+          <input type="submit" class="btncontact" value="Get It Touch" />
+        </form>
+        <div class="contact-bar">
+          <img class="end-line" src="assets/images/Shape.png" alt=" end line icon" />
+        </div>
```
Comment (levy002) [inline index.html]: - [ ] KIndly remove this line image even if it is visible on the Figma but its role is to indicate that the design is for mobile versions. Kindly remove it.

    - ### Your design
   image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @ahmadsarfrazx   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmadsarfrazx** [comment]
- Removed the icon bar from the bottom of the contact page
- Increase font-weight to 400 to match the Figma template
- Added background image in the image folder

File: style.css [hunk 4]
```diff
@@ -351,3 +352,89 @@ h2 {
   align-items: flex-start;
   gap: 15px;
 }
+
+.contact-form {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  background-image: url(assets/images/background2.png);
```
Comment (ha-manel) [inline style.css]: - [ ] Kindly add the background image like the one used in the template you chose, please use the correct path to the image (background2.png doesn't exist) and you have a background image from another template in you assets folder 👍 

## Your design:
image

## Figma design:
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @ahmadsarfrazx 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmadsarfrazx** [comment]
I have added background images in the "main" and "contact-form" branches. Background image is perfectly working in my live server.

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED  
 ## Hi @ahmadsarfrazx ,
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Your repo is professional :white_check_mark:
 * Well structured files :white_check_mark:
 * No Linter errors :white_check_mark:
 
 Now you can merge this branch and move on :1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers four_leaf_clover:


 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**